include/clang/AST/Decl.h

```diff
@@ -1584,8 +1582,6 @@ class FunctionDecl : public DeclaratorDecl, public DeclContext,
   /// 'enum Y' in 'void f(enum Y {AA} x) {}'.
   ArrayRef<NamedDecl *> DeclsInPrototypeScope;
 
-  BoundsExpr *ReturnBoundsExpr;
```

Comment:
>ReturnBoundsExpr [](start = 14, length = 16)

I'm not sure where it would go, but it might be helpful to have a comment somewhere explain the intended meaning of 'Bounds' for FunctionDecls--that it's intended to describe the return value of the function.